cmd/dynamodb/main.go

```diff
@@ -84,6 +88,40 @@ func main() {
 	lambda.Start(Handler)
 }
 
+const cutoverTime = "2024-07-12T16:10:00-07:00"
+
+// Facilitates cutting over to a new DDB stream. Before the time all
+// records will be processed in uw1. After the time, all records will be
+// processed in uw2. Making the assumption that it is extremely unlikely
+// that a record will have an exact timestamp down to the nanosecond of
+// the cutover time, so skipping that edge case. If you are reading this
+// after the cutover time you can safely revert
+// https://github.com/Clever/ddb-to-es/pull/78
+func skipRecord(record events.DynamoDBEventRecord) (bool, error) {
+	// only do this in prod and clever-dev
+	env := os.Getenv("_DEPLOY_ENV")
```

Comment:
```suggestion
	env := os.Getenv("DEPLOY_ENV")
```

AWS doesn't allow lambdas to have env vars beginning with underscore, so we don't use it. See e.g https://us-west-1.console.aws.amazon.com/cloudformation/home?region=us-west-1#/stacks/template?filteringText=ddb-to-es&filteringStatus=active&viewNested=true&stackId=arn%3Aaws%3Acloudformation%3Aus-west-1%3A589690932525%3Astack%2Fproduction--ddb-to-es%2F4729e840-1035-11e8-92d3-500cadc9fefe

Reply:
Oh, that explains why this wasnt working in my testing on Friday. I'll make this change then test again today before deploying.